FSharpLu.Azure/AzureVM.fs

```diff
@@ -1054,8 +1055,10 @@ let tryDeleteManagedImage (context:Context.InfrastructureContext) (imageResource
                 TraceTags.info "Deletion of managed image completed." tags
             return image
         with
-        | _  as e ->
-            return TraceTags.failwithException e "Failed deleting a managed image" tags
+        | :? CloudException as ex ->
+            return TraceTags.failwithException ex "Failed deleting a managed image." (tags @ ["exception", sprintf "%A" ex])
+        | e ->
+            return TraceTags.failwithException e "Failed deleting a managed image." tags
```

Comment:
Why not show the exception in the tag for both?

Reply:
I should remove the exception in the other tag as well: the exception info already get logged since this is a `failwithException` which gets logged via `trackException` in appinsight.